packages/node_modules/@ciscospark/internal-plugin-board/src/realtime-channel.js

```diff
@@ -0,0 +1,48 @@
+/**!
+ *
+ * Copyright (c) 2015-2017 Cisco Systems, Inc. See LICENSE file.
+ * @private
+ */
+
+import {
+  Mercury
+} from '@ciscospark/internal-plugin-mercury';
+
+const RealtimeChannel = Mercury.extend({
+  namespace: `Board`,
+
+  props: {
+    channelId: {
+      type: `string`,
+      required: true
+    },
+    socketUrl: {
+      type: `string`
+    },
+    binding: {
+      type: `string`
+    }
+  },
+
+  session: {
+    isSharingMercury: {
+      type: `boolean`,
+      default: false
+    },
+    socket: {
+      type: `object`
+    }
+  },
+
+  send(data) {
+    /* istanbul ignore next */
+    if (!this.socket) {
```

Comment:
Is this necessary given that the run time will throw basically the same string as an exception?